schema/cadence/schema.cql

```diff
@@ -168,6 +176,8 @@ CREATE TABLE executions (
   timer_map            map<text, frozen<timer_info>>,
   child_executions_map map<bigint, frozen<child_execution_info>>,
   request_cancel_map   map<bigint, frozen<request_cancel_info>>,
+  signal_map           map<bigint, frozen<signal_info>>,
+  signal_requested     set<uuid>,
```

Comment:
Why is this a set of uuid and signal_request_id is defined as text.
I'm assuming this is the signal_request_id kept on target execution for dedupe.

Reply:
`signal_request_id` should be uuid, good catch! changed.